generated/provider_dependencies.json

```diff
@@ -56,7 +56,8 @@
   "apache.beam": {
     "deps": [
       "apache-airflow>=2.6.0",
-      "apache-beam>=2.47.0"
+      "apache-beam>=2.53.0",
+      "pyarrow>=14.0.1"
```

Comment:
I wonder if we need to use a lower version and just excluding 14.0.0

Reply:
Well. It's the same in effect - I think we have a number of places where we directly start with >=x.y.z where z is not  0 . I think it's quite artifficial to have .0 in here. 

Unless you wanted to allow < 14.0.0, but this we actually should NOT do because the CVE affects basically all version before 0.14.1 :D 

The specification is a little misleading on that one:

Affected versions: `>= 0.14.0, < 14.0.1`

I thought initialy that only version 14.0.0 is affected, but only after closer look I noticed 0.14 not 14.0 ....

Maybe you've fallen a victim of the same assumption :D ?

Reply:
> Unless you wanted to allow < 14.0.0, but this we actually should NOT do because the CVE affects basically all versions before 14.0.1 :D

Yes, I'm not sure if all the packages supports v14. But yeah, I just rechecked the CVE, it's `>= 0.14.0, < 14.0.1` and not `>=14.0.0, < 14.0.1` :facepalm: